src/components/Home/HomeNews/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from 'react';
+import { Text, View } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import HomeNewsItem from '../HomeNewsItem';
+import styles from './styles';
+
+/* eslint-disable react/no-array-index-key */
+
+const MAX_STATIC_NEWS_COUNT = 2;
+
+export interface NewsItem {
+  datePosted: string;
+  link: string;
+  article: string;
+  excerpt: string;
+  title: string;
+  author: string;
+}
+
+const getNewsItems = async () => {
+  try {
+    const res = await fetch(
+      'https://3vmaot03ul.execute-api.us-west-1.amazonaws.com/articles',
+    );
+    if (res.status === 200) {
+      const data = await res.json();
+      return data;
+    }
+    return null;
+  } catch (err) {
+    console.error(`Error getting news items: ${err}`);
+    return null;
+  }
+};
+
+const HomeNews = ({ navigation }: any) => {
+  const [news, setNews] = useState<NewsItem[]>([]);
+  const recentNewsCount = news.length;
+  const canDisplayViewAll = recentNewsCount > MAX_STATIC_NEWS_COUNT;
+  const itemsToRender = MAX_STATIC_NEWS_COUNT;
+
+  useEffect(() => {
+    getNewsItems().then(newsItems => {
+      const sorted = newsItems.Items.sort((a: NewsItem, b: NewsItem) =>
+        Date.parse(a.datePosted) > Date.parse(b.datePosted) ? 1 : -1,
```

Comment:
What should we do if the dates are the same? I'm not sure the order will always be consistent

Reply:
Good point. An option would be to have the -1 case turn into a lexicographic comparison of the item's title.

Reply:
@wllmwu just made changes according to Bobby's suggestion!